apps/desktop-wallet/src/components/TransactionList.tsx

```diff
@@ -144,85 +146,75 @@ const TransactionList = ({
-            />
-          </TableRow>
-        ))}
-        {displayedConfirmedTxs.map((tx) => (
-          <TableRow
-            key={`${tx.hash}-${tx.address.hash}`}
-            role="row"
-            tabIndex={0}
-            onClick={() => setSelectedTransaction(tx)}
-            onKeyDown={() => setSelectedTransaction(tx)}
-          >
-            <TransactionalInfo
-              transaction={tx}
-              addressHash={tx.address.hash}
-              showInternalInflows={hideFromColumn}
-              compact={compact}
-            />
+    <Table className={className} minWidth="500px">
+      {!hideHeader && (
+        <TableHeader title={title ?? t('Transactions')}>
+          {headerExtraContent}
+          {limit !== undefined && (
+            <ActionLinkStyled onClick={() => navigate('/wallet/transfers')} Icon={ChevronRight} withBackground>
+              {t('See more')}
+            </ActionLinkStyled>
+          )}
+        </TableHeader>
+      )}
+      {stateUninitialized && (
+        <>
+          <TableRow>
+            <SkeletonLoader height="37.5px" />
           </TableRow>
-        ))}
-        {limit === undefined && (
-          <TableRow role="row">
-            <TableCell align="center" role="gridcell">
-              {allTxsLoaded ? (
-                <span>{t('All transactions loaded!')}</span>
-              ) : userAttemptedToLoadMoreTxs ? (
-                <Spinner size="15px" />
-              ) : (
-                <ActionLink onClick={handleShowMoreClick}>{t('Show more')}</ActionLink>
-              )}
-            </TableCell>
+          <TableRow>
+            <SkeletonLoader height="37.5px" />
           </TableRow>
-        )}
-        {!stateUninitialized && !pendingTxs.length && !displayedConfirmedTxs.length && (
-          <TableRow role="row" tabIndex={0}>
-            <TableCellPlaceholder align="center">{t('No transactions to display')}</TableCellPlaceholder>
+          <TableRow>
+            <SkeletonLoader height="37.5px" />
           </TableRow>
-        )}
-      </Table>
-      <ModalPortal>
-        {selectedTransaction && (
-          <TransactionDetailsModal
-            transaction={selectedTransaction}
-            onClose={() => setSelectedTransaction(undefined)}
+        </>
+      )}
+      {pendingTxs.map((tx) => (
+        <TableRow key={tx.hash} blinking role="row" tabIndex={0}>
+          <TransactionalInfo
+            transaction={tx}
+            addressHash={tx.address.hash}
+            showInternalInflows={hideFromColumn}
+            compact={compact}
           />
-        )}
-      </ModalPortal>
-    </>
+        </TableRow>
+      ))}
+      {displayedConfirmedTxs.map((tx) => (
+        <TableRow
+          key={`${tx.hash}-${tx.address.hash}`}
+          role="row"
+          tabIndex={0}
+          onClick={() => openTransactionDetailsModal(tx.hash, tx.address.hash)}
+          onKeyDown={() => openTransactionDetailsModal(tx.hash, tx.address.hash)}
```

Comment:
Should we filter the key so that it's only opening the modal when pushing enter?

Reply:
Fixed